### PR TITLE
Specify the garbage collector as the CMS

### PR DESCRIPTION
**Pull request by aloneboat**
1 .Refer to this https://github.com/alibaba/nacos/pull/10343
2.Fix README format error about MYSQL_SERVICE_DB_PARAM row

### CODE REVIEW

**Comment by aloneboat** [comment]
@paderlol  please help to review.

File: build/bin/docker-startup.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ function print_servers() {
 # JVM Configuration
 #===========================================================================================
 if [[ "${MODE}" == "standalone" ]]; then
-
+JAVA_OPT="${JAVA_OPT} -XX:+UseConcMarkSweepGC -XX:+UseCMSCompactAtFullCollection -XX:CMSInitiatingOccupancyFraction=70 -XX:+CMSParallelRemarkEnabled -XX:SoftRefLRUPolicyMSPerMB=0 -XX:+CMSClassUnloadingEnabled -XX:SurvivorRatio=8  -XX:-UseParNewGC"
   JAVA_OPT="${JAVA_OPT} -Xms${JVM_XMS} -Xmx${JVM_XMX} -Xmn${JVM_XMN}"
   JAVA_OPT="${JAVA_OPT} -Dnacos.standalone=true"
```
Comment (wilsonwu) [inline build/bin/docker-startup.sh:36]: Could you please keep same indent, thanks.

Comment (aloneboat) [reply build/bin/docker-startup.sh]: Oh，that's too bad. I fix it.

Comment (aloneboat) [reply build/bin/docker-startup.sh]: I looked at it again.I didn't mean to write on that line.so bad. Dockerfile based on jdk8，I want to specify this parameter for both standalone and other

**Pull request merged**